bin/lint

```diff
@@ -2,31 +2,21 @@
 
 set -e
 
-if [ "$TRAVIS" == "true" ]; then
-    echo "running under travis, fetching master"
-    TARGET_BRANCH=$TRAVIS_BRANCH
-    git remote set-branches --add origin $TARGET_BRANCH
-    git fetch
-else
-    TARGET_BRANCH=master
+if [ $# -ne 0 ]; then
+  if [ "$1" == "staged" ]; then
+    files=`git status --short */{src,specs} | egrep '^[^D|^R].*(jsx?)$' | awk '{ print $2 }'`
+    if [ ! -z "$files" ]; then
+      $(npm bin)/eslint $2 $files
+    fi
+  else
+    $(npm bin)/eslint --fix $1/{src,specs}
+  fi
+  exit $?
 fi
 
-if [ $# -eq 0 ]; then
-    $(npm bin)/eslint {tutor,shared,exercises}
-    exit $?
+if [[ ! -z "${OX_PROJECT}" ]]; then
+   $(npm bin)/eslint --fix $OX_PROJECT/{src,specs}
+   exit $?
 fi
```

Comment:
Wouldn't `--fix` for Travis cause it to pass when we'd want it to fail so it can be committed?

Reply:
yep, good catch.  I think I'd copied the command I was running to generate this PR from.  I'll remove that